packages/react/src/components/RadioGroup/index.tsx

```diff
@@ -107,6 +110,19 @@ const RadioGroup = ({
           aria-hidden="true"
           onClick={() => onRadioClick(index)}
         />
+        {labelDescription && (
+          <>
+            <div className="Field__break"></div>
```

Comment:
Why the extra element? Why not just apply `flex-basis: 100%` to the description?

Reply:
i could try that, was having a heck of a time getting the wrap to work properly.

Reply:
that works, pushing that change now